app/main.py

```diff
@@ -1,8 +1,45 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self,
```

Comment:
Having self on the same line with the `__init__` statement could be an option, however, consider placing `self,` on a new line. The same relates to `) -> None:` on line 5 and other `__init__` on line 12. It is more conventional and a common way across Python developers.